airbyte-webapp/src/core/domain/job/Job.ts

```diff
@@ -22,6 +59,8 @@ export interface Attempt {
   endedAt: number;
   bytesSynced: number;
   recordsSynced: number;
+  totalStats: TotalStats;
+  failureSummary: FailedSummary;
```

Comment:
Looking at the [specification](https://github.com/airbytehq/airbyte/blob/master/airbyte-api/src/main/openapi/config.yaml#L3196-L3199) both of those values are not required, and thus could be `undefined` (which should be reflected in the typing here).

Reply:
I was searching on how to get this. I think would be better if we can have the same info on this page also https://airbyte-public-api-docs.s3.us-east-2.amazonaws.com/rapidoc-api-docs.html#post-/v1/workspaces/create.

Reply:
Oh yeah, that would be great.